redesign/blog.html

```diff
@@ -20,7 +20,7 @@
     <link rel="stylesheet" href="https://unpkg.com/tachyons@4.7.0/css/tachyons.min.css"/>
     <link rel="stylesheet" href="css/dwyl.css">
     <link rel="shortcut icon" href="/img/favicon.ico">
-    <title>dwyl</title>
+    <title>Web development blog - dwyl</title>
```

Comment:
dwyl | The Blog | Web Development, Agile Methodologies and more

Reply:
@markwilliamfirth Did you see @finnhodgkin's comment above? https://github.com/dwyl/dwyl-site/pull/347#issuecomment-330883526